delete.php

```diff
@@ -0,0 +1,15 @@
+<?php
+require "db_connection.php";
+
+$question_id = $_POST['questions_id'];
+
+$sql = 'DELETE FROM questions WHERE id = :id';
+$stmt = $db->prepare($sql);
+$stmt->execute([':id'=>$_POST['questions_id']]);
```

Comment:
4行目で変数に詰めているので折角なのでそっちを使ってあげましょう。
```suggestion
$stmt->execute([':id'=>$question_id]);
```